templates/people.html

```diff
@@ -95,6 +98,18 @@ <h2>
             Members<a class="anchor-link" href="#org-members">#</a>
         </h2>
         {% endif %}
+
+        {% if section.title == "Community Members" %}
+        <div class="people-section-description media-content">
+        <p>
+            People from the Bevy community, who are not part of the organization.
```

Comment:
```suggestion
            Members of the Bevy community who are not part of the GitHub organization.
```

Reply:
I specifically did not want to use the word "members" here, to avoid confusion with org members, who are listed under a heading "Members".

Also, everywhere else on the page, we call it "The Bevy organization", and it lists all the people in the GH org anyway, without calling it that. Calling it "the GitHub organization" here feels weird and confusing.

My goal with this blurb of text was to make this "Community" part of the page clearly distinguished from the "Org" part of the page above it. The two sections not being distinct enough was what caused my initial confusion and motivated me to make this PR.